src/misc_gui.cpp

```diff
@@ -961,21 +975,63 @@ struct QueryStringWindow : public Window
 			case WID_QS_CANCEL:
 				this->Close();
 				break;
+
+			case WID_QS_MOVE: // move
```

Comment:
Most obvious comment award 😄 

```suggestion
			case WID_QS_MOVE:
```

Reply:
may have been copy/pasting from somewhere else where each case was commented lol